src/mir/passes.md

```diff
@@ -1,102 +1,169 @@
-# MIR passes
+# MIR queries and passes
 
-If you would like to get the MIR for a function (or constant, etc),
-you can use the `optimized_mir(def_id)` query. This will give you back
-the final, optimized MIR. For foreign def-ids, we simply read the MIR
+If you would like to get the MIR:
+
+- for a function - you can use the `optimized_mir(def_id)` query;
```

Comment:
there's also the MIR that const eval sees for a function: `mir_for_ctfe(def_id)`